tests/Library/Vanilla/Embeds/EmbedManagerTest.php

```diff
@@ -189,6 +191,32 @@ public function provideRenderedData() {
 </iframe>
 </div>'
             ],
+            [
+                [
+                    "url" => "https://www.twitch.tv/videos/276279462",
+                    "type" => "twitch",
+                    "name" => "20k Fortnite Friday Duos with @hysteria | 2 MINUTE STREAM DELAY",
+                    "body" => null,
+                    "photoUrl" => "https://static-cdn.jtvnw.net/s3_vods/8a24223c5b12ff7427a8_ninja_29190875424_893099877/thumb/thumb0-640x360.jpg",
+                    "height" => 281,
+                    "width" => 500,
+                    "attributes" => [
+                        "videoID" => "276279462",
+                        "embedUrl" => "https://player.twitch.tv/?video=v276279462",
+                    ],
+                ],
+'<div class="embed-video embed embedVideo">
+    <div class="embedVideo-ratio" style="padding-top: 56.2%;">
+        <button type="button" data-url="https://player.twitch.tv/?video=v276279462" aria-label="20k Fortnite Friday Duos with @hysteria | 2 MINUTE STREAM DELAY" class="embedVideo-playButton iconButton js-playVideo" style="background-image: url(https://static-cdn.jtvnw.net/s3_vods/8a24223c5b12ff7427a8_ninja_29190875424_893099877/thumb/thumb0-640x360.jpg);" title="20k Fortnite Friday Duos with @hysteria | 2 MINUTE STREAM DELAY">
+            <svg class="embedVideo-playIcon" xmlns="http://www.w3.org/2000/svg" viewBox="-1 -1 24 24">
```

Comment:
Could we pull this SVG string into a variable? It's kind of ridiculous to keep repeating it.